composer.json

```diff
@@ -15,7 +15,7 @@
         }
     ],
     "require": {
-        "php": "^7.4|^8.0",
+        "php": "^8.0",
```

Comment:
@driesvints Shouldn't this change be made in all Illuminate components `composer.json` too?

Reply:
Yep seems I forgot to do that. Can you send in a PR? Otherwise I'll try to do that later this week.

Reply:
Sure. I'll send one later today.